tictactoe/tictactoe.js

```diff
@@ -0,0 +1,94 @@
+var origBoard;
+var player;
+const player1   = "X";
+const player2   = "O";
+const winCombos = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [2,4,6],
+]
+const cells = document.querySelectorAll(".cell");
+startGame();
+
+function startGame() {
+    document.querySelector("#endgame").style.display = "none";
+    document.querySelector("#restart").style.display = "none";
+    player = player1 ;
+    document.getElementById("player").innerHTML = "X" ;
+    origBoard = Array.from(Array(9).keys());
+    for (var i = 0; i < cells.length; i++) {
+        cells[i].innerText = "";
+        cells[i].style.removeProperty("background-color");
+        cells[i].addEventListener("click", turnClick, false);
+    }
+}
+
+function turnClick(square) {
+    if (square.target.innerText == "") {
+        turn(square.target.id, player)
+        player == player1 ? player = player2 : player = player1;
+        player == player1 ?
+        document.getElementById("player").innerHTML = "X" :
```

Comment:
Values here should be based on `player1` and `player2` so you only define "X" and "Y" in a single place. DRY. This allows you to change to "A" and "B" if you want with trivial effort.

Reply:
How should I go about doing that?

Reply:
Just use `player1` instead of "X"